src/appmixer/ai/openrouter/SendPrompt/SendPrompt.js

```
@@ -0,0 +1,33 @@
+                    name: 'system'
+                },
+                {
+                    role: 'user',
+                    content: prompt,
+                    name: 'user'
+                }
+            ]
+        });
+
+        let answer = '';
+        if (data && data.choices) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)